libsrc/occ/occgenmesh.cpp

```diff
@@ -1291,7 +1291,7 @@ namespace netgen
       if (mparam.perfstepsstart <= MESHCONST_ANALYSE)
       {
         // delete mesh;
-        // mesh = make_shared<Mesh>();
+         mesh = make_shared<Mesh>();

```

Comment:
We need to preserve the given pointer in order to visualize the mesh during mesh generation. I assume the following works for you:
```cpp
if(mesh.get() == nullptr)
  mesh = make_shared<Mesh>();
```

Reply:
yes, I have experimented with exactly this and it works for us. It seems this is the missing part which makes netgen again usable for FreeCAD.
Thanks for all the work!